client/blocks/jitm/templates/modal-style.scss

```diff
@@ -121,30 +121,35 @@ $wpcom-modal-footer-height: 30px + ( $wpcom-modal-footer-padding-v * 2 );
 	@media ( min-width: $wpcom-modal-breakpoint ) {
 		flex-direction: column;
 		justify-content: flex-end;
-		max-width: 60%;
 		min-height: $wpcom-modal-content-min-height;
 		bottom: 0;
+		padding: 0 2% 0 7%;
 	}
 }
 
 .modal__text,
 .modal__visual {
 	@media ( min-width: $wpcom-modal-breakpoint ) {
 		flex: 1 auto;
-		min-width: 290px;
 	}
 }
 
 .modal__text {
-	padding: 1em 2em 1em 0;
+	padding: 1em 0;
 	margin: 0;
 	font-family: $sans;
 	font-size: $font-body-small;
+	display: flex;
```

Comment:
At certain browser width, the button ends up rendering inline with the text:

<img width="604" alt="Screen Shot 2022-07-07 at 12 07 06 AM" src="https://user-images.githubusercontent.com/104910361/177688507-e79fad32-8d57-4c3c-96e7-e83f3d209804.png">

This ensures the button does not render with text wrapped around it.